parm/use_cases/met_tool_wrapper/GridStat/GridStat_comments.conf

```diff
@@ -0,0 +1,250 @@
+# GridStat METplus Configuration
```

Comment:
I'm curious about this use case template. Is this being preserved as a demonstration of the "old way" of detailing config files? Or is it going to be used in a different setting?

Reply:
This file was accidentally left in place. It was only created (renamed from the original) to easily show the proposed changes to solicit feedback. I will remove it now.